### PR TITLE
LPS-133423 | master

### PR DESCRIPTION
**Pull request by ozysouza**
https://issues.liferay.com/browse/LPS-133423
cc @Tim-Cao

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/outoftheboxfragments/contentdisplay/ContentDisplayWithStaging.testcase [hunk 1]
```diff
@@ -101,4 +101,92 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-127548. View the selected web content just is shown after Published to live."
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/outoftheboxfragments/contentdisplay/ContentDisplayWithStaging.testcase:104]: The changes of web content won't be shown in Content Display in live site until publish to live

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/outoftheboxfragments/contentdisplay/ContentDisplayWithStaging.testcase [hunk 2]
```diff
@@ -101,4 +101,92 @@ definition {
...
+	@priority = "4"
+	test ViewContentMustBeShownJustAfterPublishedToLive {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/outoftheboxfragments/contentdisplay/ContentDisplayWithStaging.testcase:106]: PublishEditedWebContentToLive

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/outoftheboxfragments/contentdisplay/ContentDisplayWithStaging.testcase [hunk 3]
```diff
@@ -101,4 +101,92 @@ definition {
...
+				webContentTitle = "Web Content Title");
+
+			PageEditor.clickPublish();
+		}
+
+		task ("Publish to Live via page") {
+			Navigator.gotoStagedSitePage(
+				pageName = "Test Page Name",
+				siteName = "Test Site Name");
+
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/outoftheboxfragments/contentdisplay/ContentDisplayWithStaging.testcase:147]: Add step to view displayed content
```
				AssertTextEquals(
                                          locator1 = "WCD#WEB_CONTENT_CONTENT",
                                          value1 = "Web Content Content");
```

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/outoftheboxfragments/contentdisplay/ContentDisplayWithStaging.testcase [hunk 4]
```diff
@@ -101,4 +101,92 @@ definition {
...
+			Staging.gotoPublishToLive();
+
+			Staging.publishToLive();
+		}
+
+		task ("View the content is shown at view mode in staging and live site") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/outoftheboxfragments/contentdisplay/ContentDisplayWithStaging.testcase:153]: Only test one point in this task "View the displayed content at view mode in live site"

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/outoftheboxfragments/contentdisplay/ContentDisplayWithStaging.testcase [hunk 5]
```diff
@@ -101,4 +101,92 @@ definition {
...
+					pageName = "Test Page Name",
+					siteName = "${siteNameList}");
+
+				AssertTextEquals(
+					locator1 = "WCD#WEB_CONTENT_CONTENT",
+					value1 = "Web Content Content");
+			}
+		}
+
+		task ("Edit the Web Content") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/outoftheboxfragments/contentdisplay/ContentDisplayWithStaging.testcase:165]: Edit the web content in staging site

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/outoftheboxfragments/contentdisplay/ContentDisplayWithStaging.testcase [hunk 6]
```diff
@@ -101,4 +101,92 @@ definition {
...
+			WebContentNavigator.openWebContentAdmin(siteURLKey = "test-site-name-staging");
+
+			WebContentNavigator.gotoEditCP(webContentTitle = "Web Content Title");
+
+			WebContent.editCP(
+				webContentContentEdit = "Web Content Content Edit",
+				webContentTitleEdit = "Web Content Title Edit");
+		}
+
+		task ("View the edited content is shown in Staging and the old version in live site") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/outoftheboxfragments/contentdisplay/ContentDisplayWithStaging.testcase:175]: Split task to two, one is for staging site, the other one is for live site

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/outoftheboxfragments/contentdisplay/ContentDisplayWithStaging.testcase [hunk 7]
```diff
@@ -101,4 +101,92 @@ definition {
...
+					pageName = "Test Page Name",
+					siteName = "${siteNameList}");
+
+				AssertTextEquals(
+					locator1 = "WCD#WEB_CONTENT_CONTENT",
+					value1 = "${content}");
+
+				var content = "Web Content Content";
+			}
+		}
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/outoftheboxfragments/contentdisplay/ContentDisplayWithStaging.testcase:189]: Add step to Publish to Live via page in staging site

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/outoftheboxfragments/contentdisplay/ContentDisplayWithStaging.testcase [hunk 8]
```diff
@@ -101,4 +101,92 @@ definition {
...
+	}
+
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/outoftheboxfragments/contentdisplay/ContentDisplayWithStaging.testcase:191]: Add step to view edited content shown in live site

**Comment by ozysouza** [comment]
Hi @Tim-Cao  PR updated following your comments.

**Comment by Tim-Cao** [comment]
Hi @ozysouza, see changes https://github.com/liferay-echo/liferay-portal/pull/4976/commits/296727ed87581ef66ce91d636ce67c806f318320
https://github.com/liferay-echo/liferay-portal/pull/4976

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**